genesis-programs/src/non_circulating_accounts.rs

```diff
@@ -0,0 +1,30 @@
+pub(crate) const NON_CIRCULATING_ACCOUNTS: [&str; 25] = [
+    "9huDUZfxoJ7wGMTffUE7vh1xePqef7gyrLJu9NApncqA",
+    "GK2zqSsXLA2rwVZk347RYhh6jJpRsCA69FjLW93ZGi3B",
+    "HCV5dGFJXRrJ3jhDYA4DCeb9TEDTwGGYXtT3wHksu2Zr",
+    "25odAafVXnd63L6Hq5Cx6xGmhKqkhE2y6UrLVuqUfWZj",
+    "14FUT96s9swbmH7ZjpDvfEDywnAYy9zaNhv4xvezySGu",
```

Comment:
Try `solana_sdk::declare_id!()` to produce a `Pubkey` at compile time

Reply:
Hmm, can you point me at an example that would work for this case?
I think I would need to a different macro that can handle multiple keys to produce an `ids()` method...?

Reply:
You're right, it won't work out of the box.   A `pubkey!("25odAafVXnd63L6Hq5Cx6xGmhKqkhE2y6UrLVuqUfWZj")` macro that produces a Pubkey struct at compile time?